delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/OrganizationsRoutes.scala

```diff
@@ -64,39 +92,48 @@ final class OrganizationsRoutes(identities: Identities, organizations: Organizat
             // SSE organizations
             (pathPrefix("events") & pathEndOrSingleSlash) {
               operationName(s"$prefixSegment/orgs/events") {
-                lastEventId { offset =>
-                  completeStream(organizations.events(offset))
+                authorizeFor(AclAddress.Root, events.read).apply {
```

Comment:
We shouldn't need the apply

Reply:
The compiled doesn't seem to be happy when I remove `.apply`.

Reply:
Yes...we can get rid of the apply but adding some convoluted logic that Akka Http uses all over the place. I don't think is worth it in this case.